_includes/blog/tags-menu.md

```diff
@@ -0,0 +1,10 @@
+<ul id="tags-side-menu">
```

Comment:
This is an unused gutter menu. It would show  when any of the blog pages are at `899px` or up.
It shows the tags. NOTE: This can be removed if you don't want it in the code base.